debug.log

```diff
@@ -0,0 +1,12 @@
+[0324/223225.836:ERROR:process_reader_win.cc(123)] NtOpenThread: {Access Denied} A process has requested access to an object, but has not been granted those access rights. (0xc0000022)
+[0324/223225.836:ERROR:process_reader_win.cc(123)] NtOpenThread: {Access Denied} A process has requested access to an object, but has not been granted those access rights. (0xc0000022)
+[0324/223225.836:ERROR:process_reader_win.cc(123)] NtOpenThread: {Access Denied} A process has requested access to an object, but has not been granted those access rights. (0xc0000022)
+[0324/223225.884:ERROR:exception_snapshot_win.cc(88)] thread ID 8224 not found in process
+[0324/223225.884:ERROR:exception_snapshot_win.cc(88)] thread ID 20052 not found in process
+[0324/223225.884:WARNING:crash_report_exception_handler.cc(62)] ProcessSnapshotWin::Initialize failed
+[0324/223225.884:ERROR:exception_snapshot_win.cc(88)] thread ID 20056 not found in process
+[0324/223225.884:WARNING:crash_report_exception_handler.cc(62)] ProcessSnapshotWin::Initialize failed
+[0324/223225.885:WARNING:crash_report_exception_handler.cc(62)] ProcessSnapshotWin::Initialize failed
+[0324/223226.568:ERROR:process_reader_win.cc(123)] NtOpenThread: {Access Denied} A process has requested access to an object, but has not been granted those access rights. (0xc0000022)
+[0324/223226.621:ERROR:exception_snapshot_win.cc(88)] thread ID 19572 not found in process
+[0324/223226.621:WARNING:crash_report_exception_handler.cc(62)] ProcessSnapshotWin::Initialize failed
```

Comment:
I don't think this file is meant to be committed?

Reply:
Yes that's right. My bad. I have removed it.